packages/sdk/ts/browser/index.ts

```diff
@@ -1,28 +1,13 @@
-export * from "../verifyingKeys";
-export * from "../maci";
-export * from "../relayer";
 export * from "../poll";
-export * from "../proof";
+export * from "../proof/download";
+export * from "../proof/types";
 export * from "../tally";
 export * from "../trees";
 export * from "../vote";
-export * from "../utils";
-export * from "../user";
 export * from "../maciKeys";
-export {
-  EMode,
-  EContracts,
-  EPolicies,
-  EInitialVoiceCreditProxies,
-  extractVk,
-  genProofSnarkjs,
-  formatProofForVerifierContract,
-  verifyProof,
-  cleanThreads,
-  unlinkFile,
-  getBlockTimestamp,
-  genEmptyBallotRoots,
-} from "@maci-protocol/contracts";
+export { getSignedupUserData, signup, getJoinedUserData, hasUserJoinedPoll, hasUserSignedUp } from "../user";
```

Comment:
Let's use `export * from "../user";` as for other exports

Reply:
we don't want to export joinPoll from `../user` because it relies on maci-contracts functions

Reply:
I made a separate version of `joinPoll` that uses wasm specifically and functions implemented from within maci-sdk directly (inside ts/browser/)